indexing-service/src/main/java/org/apache/druid/indexing/seekablestream/SeekableStreamIndexTaskRunner.java

```diff
@@ -1058,7 +1058,8 @@ private synchronized void persistSequences() throws IOException
                 ingestionState,
                 getTaskCompletionUnparseableEvents(),
                 getTaskCompletionRowStats(),
-                errorMsg
+                errorMsg,
+                false
```

Comment:
Why is this always `false`? Does it make more sense to be always `true` because realtime tasks will fail when handoff fails?

Reply:
I went with `errorMsg == null` and added a javadoc stating the errorMsg String should be null on success and non-null on failure. Current code follows that pattern

Reply:
Oh yeah, I thought `IngestionStatsAndErrorsTaskReportData` is available only when the task succeeded. I think you are correct.